### ISSUE TITLE
Permission problems on MacOS

### ISSUE THREAD

**Issue by BentEngbers** [ImpulseAdventure/GUIslice-Builder#213]
**Describe the bug**
I got a permission denied on mac os (M1).
```
./GUIslice.sh: line 9: /***/****/****/GUIsliceBuilder/bin/java: Permission denied
```
**To Reproduce**
1. Download the latest version
2. Unzip the directory
3. Run the `./GUIslice.sh` command

**Expected behavior**
I expect the program to start.

**Version**
- Builder version: 0.17b12
- OS: MacOS 12.6

**Workaround**
The following command can fix the problems:
```sh
chmod +x ./bin/java
```
![CleanShot 2022-10-23 at 19 07 47@2x](https://user-images.githubusercontent.com/16548374/197405793-f4f6a60c-0ec1-4331-b475-725ed0bd09b7.png)
When the executable permissions of the java binary have been changed, an additional problem is revealed:
```
****/****/GUIsliceBuilder/GUIslice.sh: line 9: 95525 Killed: 9               "$DIR/bin/java" $CDS_JVM_OPTS -p "$DIR/../app" -m builder/builder.Builder "$@"
```
Additional allowance of java is also necessary on MacOS: 
![CleanShot 2022-10-23 at 19 14 39@2x](https://user-images.githubusercontent.com/16548374/197406019-8d33c3fe-f6a9-4599-ac6a-1b4a7a4b633a.png)

**Comment by Pconti31** [comment]
@BentEngbers Thank you for the information.  I'll spin a new release setting execute permission on java.  Not much i can do about being an "identified developer".   If you don't mind I'll copy your Security & Privacy image to my readme so others will be prepared.
Paul--

**Comment by Pconti31** [comment]
fixed in release 0.17.b13
